### DISCUSSION TITLE
How is this better/different than obsidian?

### DISCUSSION THREAD

**Discussion by huachuman** [Feel-ix-343/markdown-oxide#150] [Q&A] [answered]
I'm not really seeing much other than some autocompletion features that distinguish this from the very much more feature rich and extendable obsidian. Can someone help me understand?

**Comment by Feel-ix-343** [comment, marked as answer]
Hello!

I made this so I can write my Obsidian notes in my favorite text-editor neovim. The obsidian editor was not cutting it for me, and I wanted to use the text-editor I code for hours each day with. 

This is the main distinguishing factor! Other than this, there are certain PKM features I like better about this than Obsidian, but I still use Obsidian frequently. I really like the dataview plugin and I use Obsidian sync and publish. I usually have neovim on the left split and preview with Obsidian on the right.

**Comment by huachuman** [comment]
Thanks for clarifying that is actually very helpful.

**Reply by Feel-ix-343** [reply]
No problem!

Hopefully more of these questions will be cleared up in our new website https://oxide.md :eyes: (when it develops a bit more) The site even uses Obsidian publish!

Leave comments here or open another discussion for any more questions!
